BlazarService/src/main/java/com/hubspot/blazar/guice/BlazarServiceModule.java

```diff
@@ -91,34 +88,33 @@ public void configure(Binder binder) {
     binder.bind(ObjectMapper.class).toInstance(getEnvironment().getObjectMapper());
     Multibinder.newSetBinder(binder, ContainerRequestFilter.class).addBinding().to(GitHubNamingFilter.class).in(Scopes.SINGLETON);
 
+    Multibinder<RepositoryBuildVisitor> repositoryBuildVisitorMultibinder = Multibinder.newSetBinder(binder, RepositoryBuildVisitor.class);
```

Comment:
Because we add bindings to them in multiple places. It seemed cleaner to me to move all the slack things into `BlazarSlackModule` and then pass in the Multibinders rather than to pass the slack configuration to every class that might need to bind a slack related thing (Visitors, Slack sessions etc.)